packages/appeals-service-api/src/spec/appellant-submission.yaml

```diff
@@ -3,11 +3,6 @@ components:
     AppellantSubmission:
       description: An appellant submission
       type: object
-      required:
```

Comment:
removed this to allow partial patching, will need a separate schema for the creation